src/algebra/ordered_monoid.lean

```diff
@@ -625,7 +625,7 @@ class ordered_cancel_add_comm_monoid (α : Type u)
 
 /-- An ordered cancellative commutative monoid
 is a commutative monoid with a partial order,
-in which multiplication is cancellative and strictly monotone. -/
+in which multiplication is cancellative and monotone. -/
 @[protect_proj, ancestor comm_monoid left_cancel_semigroup right_cancel_semigroup partial_order]
```

Comment:
You could put a `to_additive` here too I suppose and get rid of the `attribute` below.

Reply:
Yes, I only just learnt this trick. Thanks.